data/json/materials.json

```
@@ -1219,6 +1219,32 @@
       { "fuel": 1, "smoke": 1, "burn": 2 }
     ]
   },
+  {
+    "type": "material",
+    "ident": "mixed_fabric",
+    "name": "mixed fabric",
```

I don't know why, but we capitalize materials' names.

```suggestion
    "name": "Mixed Fabric",
```

---

Alright. Done.